README.md

```diff
@@ -246,6 +248,76 @@ expect(wrapper.find('#child')).to.have.descendants('#last')
 expect(wrapper).to.not.have.descendants('#root1')
 ```
 
+#### `descendantsCount(selector)`
+
+| render | mount | shallow |
+| -------|-------|-------- |
+| yes    | yes   | yes     |
+
+
+Assert that the wrapper contains a exact amount of descendants matching the given selector.
+Defaults to looking for exactly 1 descendent
+
+```js
+import React from 'react'
+import {mount, render, shallow} from 'enzyme'
+
+class Fixture extends React.Component {
+  render () {
+    return (
+      <div id='root'>
+        <span id='child'>
+          <span id='last'></span>
+          <span id='last'></span>
+        </span>
+      </div>
+    )
+  }
+}
+
+const wrapper = mount(<Fixture />) // mount/render/shallow when applicable
+
+expect(wrapper).to.have.descendantsCount('#root')
```

Comment:
i actually would prefer if we incorporate `length`:

```js
expect([ 1, 2, 3]).to.have.length(3);
expect('foo').to.have.length.above(2);
expect('foo').to.have.length.below(4);
expect('foo').to.have.length.within(2,4);
```

So it ends up like this:

```js
expect(wrapper).to.have.descendants('.foo').with.length(5)
```

in order to accomplish this, we need to flag the descendants:

```
flag(this, 'object', theDescendants)
```

this will make `TestWrapper#hasDescendants` pointless, instead we should just add a `find` method that delegates to enzyme's wrapper

makes sense?